core/domain/classifier/lda_string_classfier.py

```diff
@@ -0,0 +1,598 @@
+# coding: utf-8
```

Comment:
Yes, it's not completely done. If the current implementation meets our needs, I'll try to remove the original code and replace corresponding references later.